pkgs/tools/text/gpt2tc/0001-add-python-shebang.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/gpt2convert.py b/gpt2convert.py
+index 34ca909..6e6cac5 100644
+--- a/gpt2convert.py
++++ b/gpt2convert.py
+@@ -1,3 +1,5 @@
++#!/usr/bin/env python3
++
```

Comment:
this should probably be upstreamed as well.

Reply:
I can try to email Fabrice Bellard about this.